README.md

```diff
@@ -2,6 +2,35 @@
 
 A web-based diagram client framework for the [Graphical Language Server Platform (GLSP)](https://github.com/eclipse-glsp/glsp) based on [Eclipse Sprotty](https://github.com/eclipse/sprotty).
 
+## Workflow Diagram Example
+
+The workflow diagram is a consistent example provided by all GLSP components. The example implements a simple flow chart diagram editor with different types of nodes and edges (see screenshot below). The example can be used to try out different GLSP features, as well as several available integrations with IDE platforms (Theia, VSCode, Eclipse, Standalone).
+As the example is fully open source, you can also use it as a blueprint for a custom implementation of a GLSP diagram editor.
+See [our project website](https://www.eclipse.org/glsp/documentation/#workflowoverview) for an overview of the workflow example and all components implementing it.
+
+![Workflow Diagram](/documentation/standalone-diagram.gif)
+
+### How to start the Workflow Diagram example?
+
+First, you need to build the Client packages:
+
+```
+yarn install
+```
+
+And the GLSP Server (Which includes the Workflow Diagram Server example). See [`glsp-server`](https://github.com/eclipse-glsp/glsp-server#building) for instructions.
+
+Once both the Server and the Client packages are built, you can start both:
+
+- Server: in `examples/org.eclipse.glsp.example.workflow`, start `org.eclipse.glsp.example.workflow.launch.WorkflowServerLauncher.java` with these arguments: `--websocket --port=8081` (Or use the included `Start_Workflow_Example_Server_(Websocket).launch` Launch Configuration from Eclipse).
+- Client: Open the `examples/workflow-standalone/app/diagram.html` file in your favorite browser
+
+### Where to find the sources?
+
+In addition to this repository, the related source code can be found here:
+
+-   https://github.com/eclipse-glsp/glsp-server
```

Comment:
I'd prefer just mentioning the link in the paragraph above instead of having a single-bullet bullet list.

Reply:
I followed the structure for all other READMEs (The glsp-client is the only one with a single related repo); but removing the list is also fine